notifier/core/config.py

```diff
@@ -14,7 +14,10 @@ def __init__(self):
         self.__load()
 
     def key(self, name):
-        return self.__conf.get(name, None)
+        param = self.__conf.get(name, None)
```

Comment:
the `get` method returns by default `None` if the key is not found..., why do you put `None` explicitly?

Reply:
Fix de4c258